src/handlers/shared/stop.ts

```diff
@@ -13,7 +14,7 @@ export async function stop(context: Context, issue: Context["payload"]["issue"],
   if (!userToUnassign) {
     const log = logger.error("You are not assigned to this task", { issueNumber, user: sender.login });
     await addCommentToIssue(context, log?.logMessage.diff as string);
-    return { output: "You are not assigned to this task" };
+    return { content: "You are not assigned to this task", status: "ok" };
```

Comment:
```suggestion
    return { content: "You are not assigned to this task", statusCode: 200 };
```

Reply:
Although this doesn't feel like `200` would be the correct "http status code" but you get the idea.